Add XOR Model Example

JacobSzwejbka opened the pull request:
Summary: Add a real basic model/use-case to showcase the full training loop from model definition to optimizer.step()

Differential Revision: D62771102

iseeyuan reviewed (approved):
LGTM. In the following up PRs, would you add a README for instructions on how to work with those codes?

JacobSzwejbka committed:
Add XOR Model Example (#5397)

Summary:

Add a real basic model/use-case to showcase the full training loop from model definition to optimizer.step()

Reviewed By: iseeyuan

Differential Revision: D62771102
